sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -261,13 +273,14 @@ class AioHttpTransportResponse(AsyncHttpResponse):
     :param block_size: block size of data sent over connection.
     :type block_size: int
     """
-    def __init__(self, request: HttpRequest, aiohttp_response: aiohttp.ClientResponse, block_size=None) -> None:
+    def __init__(self, request, aiohttp_response, block_size=None):
+        # type: (HttpRequest, aiohttp.ClientResponse, Optional[int]) -> None
         super(AioHttpTransportResponse, self).__init__(request, aiohttp_response, block_size=block_size)
         # https://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse
-        self.status_code = aiohttp_response.status
-        self.headers = CIMultiDict(aiohttp_response.headers)
-        self.reason = aiohttp_response.reason
-        self.content_type = aiohttp_response.headers.get('content-type')
+        self.status_code = aiohttp_response.status  # type:ignore
```

Comment:
Why not use `typing.cast`? To avoid having to import typing in the first place? `# type: ignore` will type self.status_code as `typing.Any` which is most likely what you were trying to do, right?

Reply:
Want to make sure I understand correctly, do you mean:

```py
response = cast(aiohttp.ClientResponse, aiohttp_response)
```